src/skin/options.html

```diff
@@ -196,6 +196,15 @@ <h4 id="privacy-settings-header" class="i18n_options_privacy_settings" style="di
           </span>
         </label>
       </div>
+      <div class="checkbox" id="disable-network-prediction" style="display:none">
+        <label>
+          <input type="checkbox" id="disable-network-prediction-checkbox">
+          <span>
+            <span class="i18n_options_disable_network_prediction"></span>
+            <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Link_prefetching_FAQ" target="_blank"><span class="ui-icon ui-icon-circle-b-help"></span></a>
```

Comment:
May need a better help link as a lot of this one is specific to Firefox/not true in Chrome (`<link>`-initiated prefetching is NOT disabled in Chrome).

Reply:
Maybe link to https://www.google.com/intl/en/chrome/privacy/whitepaper.html#netpredict instead in Chrome-based browsers?

Reply:
Do you mean replacing the link entirely with this chrome one, or flipping which link is displayed based on the browser?

Reply:
The latter.